assets/stylesheets/shared/_component.scss

```diff
@@ -0,0 +1,6 @@
+@import 'functions';                    // functions that we've used from Compass, ported over
+@import 'functions/functions';          // sass functions for z-index, etc.
+@import 'colors';                       // import all of our wpcom colors
+@import 'typography';                   // all the typographic rules, variables, etc.
+@import 'mixins/mixins';                // sass mixins for gradients, bordius radii, etc.
+@import 'extends';                      // sass extends for commonly used styles
```

Comment:
We probably need to move more code here. We will discover what we missed when we start playing with more folders.